models/quote.js

```diff
@@ -41,7 +51,9 @@ Object.assign(quote.methods, {
 
   trackPublishing(type) {
     if (type !== 'public') return;
-    this.publishedAt = new Date;
+    if (delete this.owner) {
+      this.publishedAt = new Date;
+    }
```

Comment:
What should `#trackPublishing` do? It seems poorly named? :confused: 
Is it supposed to publish something publicly? Why not `#publish` or `#makePublic`?

Besides I'm having hard time understanding that `delete` snippet. According to [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete) `delete` operator returns:
>true for all cases except when the property is an own non-configurable property, in which case, false is returned in non-strict mode.

Which pretty much means in your case it is absolutely always true :dizzy_face: 
Putting that aside for a moment, deleting stuff is simply wrong way of doing thing. It is one of slowest V8 operations and you don't need it here. Instead simply put its value to `undefined`.